### DISCUSSION TITLE
Powerline arrows a different colour in VSCode

### DISCUSSION THREAD

**Discussion by SiJiL82** [JanDeDobbeleer/oh-my-posh#2099] [Troubleshoot] [answered]
For some reason the arrow after my OS segment is displaying as a different colour to the OS segment, but only in VSCode terminals.
Here I've got the OS segment set to #cfcfc4 but the arrow is a pure white. Strangely, if I set the OS segment to #ff0000 (so pure red) it makes the arrow slightly darker...
WSL (Ubuntu) terminal:
![image](https://user-images.githubusercontent.com/65114782/163671930-acfbb486-3985-4b12-8684-85f782f2321b.png)
PoSH:
![image](https://user-images.githubusercontent.com/65114782/163671952-1d586836-711d-4380-82f5-d7efb27d6a1e.png)

It works fine in Windows Terminal though:
![image](https://user-images.githubusercontent.com/65114782/163671962-d6d06b0e-0f22-4cc2-9cb7-8339e7eff21e.png)

Anyone know how to fix it?

**Comment by JanDeDobbeleer** [comment, marked as answer]
I do. Add the following setting:

```
"terminal.integrated.minimumContrastRatio": 1
```

See [this](https://github.com/microsoft/vscode/issues/146406) issue.

**Reply by SiJiL82** [reply]
Perfect, thanks!
